idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -661,11 +676,7 @@ def eq_velocity_liq(blk, t, x):
 
         # vapor mass transfer coeff. for diffusing components [mol/m2.s.Pa]
         def rule_mass_transfer_coeff_vap(blk, t, x, j):
-            if x == self.vapor_phase.length_domain.first():
-                return Expression.Skip
-            elif j == "MEA":
-                return Expression.Skip
-            else:
+            if j in liq_comp_diffus:
```

Comment:
This needs some work - at the moment the indexing set is based on liquid components, but the expression is based on vapor components.

First, the unit model needs to create a new indexing set: `self.equilibrium_components = self.congif.vapor_property_package & self.config.liquid_property_package`

This will give us the intersection of the two component lists - i.e. the components that cross the interface.

Then, this Expression can be indexed by `self.equilibrium_components` and there should be no need for `Expression.Skip`.

We will also be able to use this later on in other constraints and expressions.

Reply:
Yes, I agree with this.